Workleap.DotNet.CodingStandards.nuspec

```diff
@@ -14,6 +14,8 @@
     <repository type="git" url="$RepositoryUrl$" commit="$RepositoryCommit$" branch="$RepositoryBranch$" />
     <dependencies>
       <dependency id="Microsoft.CodeAnalysis.BannedApiAnalyzers" version="3.3.4" />
+      <dependency id="Microsoft.CodeAnalysis.NetAnalyzers" version="8.0.0" />
+      <dependency id="StyleCop.Analyzers" version="1.2.0-beta.556" />
```

Comment:
Hot topic to discuss:

Out of our 2 early adopters, the 2 showed interest for a rule that is NOT part of `Microsoft.CodeAnalysis.NetAnalyzers`. It was about the .cs file names matching the types.

So the question is: Do we want to re-introduce StyleCop BUT only for a VERY limited set of rules (so far, one)?

Reply:
Does it have impact on build time?

There are other analyzers providing this rule, such as Meziantou.Analyzer (self promo 😉), https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0048.md (`MA0048.only_validate_first_type = false`). Does other analyzers provide other pertinent rules in addition to this specific rule?

Adding a package increase build time and maintenance time (Renovate), so if we have the choice, we could use a package with more relevant rules.

Reply:
Nope it doesn't. See the updated PR description for statistics and details.

For your Meziantou.* packages, you should really go through the AppSec approval process.

Reply:
I will go through the AppSecc process if people think there is a need for the package. I don't want to waste their time unless it's needed.